tests/integration/helm/upgrade/n_minus_2/main_test.go

```diff
@@ -0,0 +1,29 @@
+// +build integ
+//  Copyright Istio Authors
+//
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+
+package nminus2
+
+import (
+	"testing"
+
+	"istio.io/istio/pkg/test/framework"
+)
+
+func TestMain(m *testing.M) {
```

Comment:
Are we installing the control plane twice here? Once by the test framework and again through `InstallIstio`?

Reply:
No we aren't. Because we never call `istio.Setup` it never installs Istio using the `iop.yaml`